src/backend/vulkan/src/device.rs

```diff
@@ -1404,7 +1404,7 @@ impl d::Device<B> for Device {
             d::WaitFor::All => true,
         };
         let result = unsafe {
-            self.raw.0.wait_for_fences(&fences, all, timeout_ms as u64 * 1000)
```

Comment:
was this code incorrect all the time? should have been `timeout_ms * 1_000_000`

Reply:
it seems like that